x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_inventory/service_inventory.spec.ts

```diff
@@ -53,44 +46,44 @@ describe('When navigating to the service inventory', () => {
       })
     );
   });
-
-  beforeEach(() => {
-    cy.loginAsViewerUser();
-    cy.visitKibana(serviceInventoryHref);
-  });
-
   after(() => {
     synthtrace.clean();
   });
 
-  it('has no detectable a11y violations on load', () => {
-    cy.contains('h1', 'Services');
-    // set skipFailures to true to not fail the test when there are accessibility failures
-    checkA11y({ skipFailures: true });
-  });
+  describe('When navigating to the service inventory', () => {
+    beforeEach(() => {
+      cy.loginAsViewerUser();
+      cy.visitKibana(serviceInventoryHref);
+    });
 
-  it('has a list of services', () => {
-    cy.contains('opbeans-node');
-    cy.contains('opbeans-java');
-    cy.contains('opbeans-rum');
-  });
+    it('has no detectable a11y violations on load', () => {
+      cy.contains('h1', 'Services');
+      // set skipFailures to true to not fail the test when there are accessibility failures
+      checkA11y({ skipFailures: true });
+    });
 
-  it('has a list of environments', () => {
-    cy.get('td:contains(production)').should('have.length', 3);
-  });
+    it('has a list of services', () => {
+      cy.contains('opbeans-node');
+      cy.contains('opbeans-java');
+      cy.contains('opbeans-rum');
+    });
+
+    it('has a list of environments', () => {
+      cy.get('td:contains(production)').should('have.length', 3);
+    });
 
-  it('when clicking on a service it loads the service overview for that service', () => {
-    cy.contains('opbeans-node').click({ force: true });
-    cy.url().should('include', '/apm/services/opbeans-node/overview');
-    cy.contains('h1', 'opbeans-node');
+    it('when clicking on a service it loads the service overview for that service', () => {
+      cy.contains('opbeans-node').click({ force: true });
+      cy.url().should('include', '/apm/services/opbeans-node/overview');
+      cy.contains('h1', 'opbeans-node');
+    });
   });
 
-  describe.skip('Calls APIs', () => {
+  describe('Calls APIs', () => {
     beforeEach(() => {
-      [...mainApiRequestsToIntercept, ...secondaryApiRequestsToIntercept].map(
-        ({ endpoint, aliasName }) => {
-          cy.intercept('GET', endpoint).as(aliasName);
-        }
+      cy.intercept('GET', '/internal/apm/services?*').as('servicesRequest');
+      cy.intercept('POST', '/internal/apm/services/detailed_statistics?*').as(
+        'detailedStatisticsRequest'
       );
```

Comment:
Shouldn't `mainApiRequestsToIntercept` and `secondaryApiRequestsToIntercept` be removed if you choose to go this route of splitting it up?

If not I think you can keep it like before

Reply:
But I see you still have:

```
  {
    endpoint: '/internal/apm/services/detailed_statistics?*',
    aliasName: 'detailedStatisticsRequest',
  },
```

Which will setup an interceptor for `GET /internal/apm/services/detailed_statistics` which is not a valid route